frontend/src/components/Reviews/ReviewCard.vue

```diff
@@ -28,7 +32,17 @@ export default {
     body: String,
     published: String,
     authenticated: Boolean,
-    recommend: Boolean
+    recommend: Boolean,
+    session: {
+      default: 0,
+      type: Number
+    }
+  },
+  computed: {
+    sessionShortName() {
+      return this.$store.getters.sessions.length &&
+        this.$store.getters.sessions[this.session].shortName
```

Comment:
This displays the wrong session because database id's start at 1 and array indicies start at 0